gui/fitpanel/src/TFitEditor.cxx

```diff
@@ -2803,15 +2803,19 @@ void TFitEditor::DoSliderXMoved()
 
 void TFitEditor::DrawSelection(bool restore)
 {
+   #ifndef R__HAS_COCOA
    static Int_t  px1old, py1old, px2old, py2old; // to remember the square drawn.
+   #endif
```

Comment:
To be consistent with other similar uses of preprocessor directives in this file, I would remove whitespace before `#`, i.e.
```suggestion
#ifndef R__HAS_COCOA
   static Int_t  px1old, py1old, px2old, py2old; // to remember the square drawn.
#endif
```